spec/features/checkout_spec.rb

```diff
@@ -50,12 +57,56 @@
     end
 
     scenario "I select 'Triodos Bank' it creates a transaction at Sisow and redirects me there" do
-      select 'Triodos Bank', from: :issuer_id
-      click_button 'Save and Continue'
+      select "Triodos Bank", from: :issuer_id
+      click_button "Save and Continue"
       expect(WebMock).to have_requested(:get, sisow_request_url). with(query: hash_including(sisow_request_params))
       response = page.driver.response
       expect(response.status).to be 302
       expect(response.headers["Location"]).to match(%r(https://www\.sisow\.nl/Sisow/iDeal/Simulator\.aspx\?merchantid=2537407799&txid=\w*&sha1=\w*))
     end
   end
+
+  context "site has has paymentmethod Paypal" do
+    let(:paypal) do
+      Spree::PaymentMethod::SisowBilling::Paypalec.create!(name: "Paypal")
+    end
+    let(:redirect_url_response) do
+      stored_response("paypal_redirect_url_output")
+    end
+    let(:sisow_request_params) do
+      { amount: "2000",
+        callbackurl: "http://www.example.com/sisow/#{order.number}",
+        cancelurl: "http://www.example.com/orders/#{order.number}/checkout/sisow_cancel",
+        description: "Spree Test Store - Order: #{order.number}",
+        merchantid: "2537407799",
+        notifyurl: "http://www.example.com/sisow/#{order.number}",
+        payment: "paypalec",
+        purchaseid: order.number,
+        returnurl: "http://www.example.com/orders/#{order.number}/checkout/sisow_return",
+        shop_id: '' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.